src/screens/ProfileScreen/elements/ProfilePicture.js

```diff
@@ -1,61 +1,69 @@
-import { DEFAULT_PROFILE_PIC_PATH } from '../../../utils/constants';
-import FastImage from 'react-native-fast-image';
+import {DEFAULT_PROFILE_PIC_PATH} from '../../../utils/constants';
+import {normalize} from '../../../utils/fonts';
+import {CircleGradient} from '../../../components/Karma/CircleGradient';
 
-/**
- * @typedef {Object} ProfilePicturePropsParam
- * @property {function} onImageContainerClick
- * @property {String} profilePicPath
- * @property {Boolean} disabledAddIcon
- */
-/**
- *
- * @param {ProfilePicturePropsParam} props
- */
-const ProfilePicture = ({ onImageContainerClick, profilePicPath, disabledAddIcon = false}) => {
-    const __renderAddIcon = () => {
-        if(!profilePicPath || !disabledAddIcon) return <></>
+const ProfilePicture = ({
+  onImageContainerClick,
+  profilePicPath,
+  disabledAddIcon = false,
+  karmaScore = 0
+}) => {
+  const renderAddIcon = () => {
+    if (!profilePicPath || !disabledAddIcon) return <></>;
 
-        return <MemoIcAddCircle width={48} height={48} style={styles.addCircle} />
-    }
-    return <View style={styles.wrapImageProfile}>
-        <TouchableNativeFeedback onPress={onImageContainerClick}>
-            <View style={styles.profileImageContainer}>
+    return <MemoIcAddCircle width={48} height={48} style={styles.addCircle} />;
+  };
+  return (
+    <View style={styles.wrapImageProfile}>
+      <TouchableNativeFeedback onPress={onImageContainerClick}>
+        <CircleGradient fill={karmaScore} size={normalize(100)} width={normalize(2)}>
+          <View style={styles.profileImageContainer}>
             <Image
-                style={styles.profileImage}
-                source={{
-                uri: profilePicPath
-                    ? `${profilePicPath}`
-                    : DEFAULT_PROFILE_PIC_PATH,
-                }}
-                resizeMode={FastImage.resizeMode.stretch}
+              style={styles.profileImage}
+              source={{
+                uri: profilePicPath ? `${profilePicPath}` : DEFAULT_PROFILE_PIC_PATH
+              }}
+              resizeMode={FastImage.resizeMode.stretch}
             />
-            { __renderAddIcon() }
-            </View>
-        </TouchableNativeFeedback>
+            {renderAddIcon()}
+          </View>
+        </CircleGradient>
+      </TouchableNativeFeedback>
     </View>
-}
+  );
+};
 
 let styles = StyleSheet.create({
-    addCircle: {position: 'absolute', top: 25, left: 25},
-    profileImage: {
-        width: 100,
-        height: 100,
-        borderRadius: 100,
-        // marginBottom: 12
-    },
-    profileImageContainer: {
-        width: 100,
-        borderRadius: 100,
-    },
-    wrapImageProfile: {
-        marginTop: 14,
-        flexDirection: 'column',
-    },
-})
+  addCircle: {position: 'absolute', top: 25, left: 25},
+  profileImage: {
+    width: 97,
+    height: 97,
+    borderRadius: 100,
+    marginLeft: 1.4,
+    marginTop: 2,
+    marginBottom: 2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The size and positioning of the profile image have been adjusted. While this may improve the appearance of the image, it's important to ensure that these changes don't negatively affect the layout on different screen sizes or devices. It might be beneficial to test these changes on various screen sizes to ensure consistent display.



<!-- This is an auto-generated comment by OSS CodeRabbit -->